Examples/Tests/circuits/inputs_maurice_surfaceInt

```diff
@@ -0,0 +1,211 @@
+my_constants.dz = Lz / n_cellz
+
+my_constants.tiny = 1.e-12
+
+###############
+# excitation
+###############
+
+warpx.E_excitation_on_grid_style = parse_E_excitation_grid_function
+
+warpx.Ex_excitation_flag_function(x,y,z) = "0."
+warpx.Ey_excitation_flag_function(x,y,z) = "0."
+warpx.Ez_excitation_flag_function(x,y,z) = "2* (x > -w_port/2.) * (x < w_port/2.) * (z > 0.) * (z < h_port) * (y >= -dy/2.) * (y <= dy/2.)"
+
+warpx.Ex_excitation_grid_function(x,y,z,t) = "0."
+warpx.Ey_excitation_grid_function(x,y,z,t) = "0."
+warpx.Ez_excitation_grid_function(x,y,z,t) = "sin(2*pi*freq*t) * (t < (1./freq))"
+
+warpx.B_excitation_on_grid_style = parse_B_excitation_grid_function
+
+warpx.Bx_excitation_flag_function(x,y,z) = "2* (x > -w_port/2.) * (x < w_port/2.) * (z > 0.) * (z < h_port) * (y >= 0.) * (y <= dy)"
+warpx.By_excitation_flag_function(x,y,z) = "0."
+warpx.Bz_excitation_flag_function(x,y,z) = "0."
+
+warpx.Bx_excitation_grid_function(x,y,z,t) = "(mu_0*mu_r_SiO2/50.)*(sin(2*pi*freq*t)) * (t < (1./freq))"
+warpx.By_excitation_grid_function(x,y,z,t) = "0."
+warpx.Bz_excitation_grid_function(x,y,z,t) = "0."
+
+###############
+# diagnostics
+###############
+
+diagnostics.diags_names = plt
+#diagnostics.diags_names = plt input output
+
+###############
+# full plotfiles
+plt.intervals = 10
+# plt.fields_to_plot = Ez Bx
+plt.fields_to_plot = Ex Ey Ez Bx By Bz epsilon sigma mu
+#plt.fields_to_plot = Ez epsilon sigma mu
+plt.diag_type = Full
+
+########################## Surface Integral on the Y-Plane #############################
+my_constants.eps=1.e-9
+
+warpx.reduced_diags_names = Int_yInputPlane Int_yIntermPlane Int_yOutputPlane
+Int_yInputPlane.type = RawEFieldReduction
+Int_yInputPlane.reduction_type = integral
+Int_yInputPlane.integration_type = surface
+Int_yInputPlane.intervals = 1
+Int_yInputPlane.surface_normal = Y
+my_constants.YInputPlaneLoc=2*dy #Nodes aligned with Y
+
+Int_yIntermPlane.type = RawEFieldReduction
+Int_yIntermPlane.reduction_type = integral
+Int_yIntermPlane.integration_type = surface
+Int_yIntermPlane.intervals = 1
+Int_yIntermPlane.surface_normal = Y
+my_constants.YIntermPlaneLoc=Ly - l_Al #Nodes aligned with Y
+
+Int_yOutputPlane.type = RawEFieldReduction
+Int_yOutputPlane.reduction_type = integral
+Int_yOutputPlane.integration_type = surface
+Int_yOutputPlane.intervals = 1
+Int_yOutputPlane.surface_normal = Y
+my_constants.YOutputPlaneLoc=Ly-2*dy #Nodes aligned with Y
+
+ Int_yInputPlane.reduced_function(x,y,z) = " (x >= -w_port/2.) * (x <= w_port/2.) * (y >= YInputPlaneLoc - dy/2.) * (y <= YInputPlaneLoc) * (z > 0.) * (z <= h_port) * 1"
+Int_yIntermPlane.reduced_function(x,y,z) = " (x >= -w_port/2.) * (x <= w_port/2.) * (y >= YIntermPlaneLoc - dy/2.) * (y <= YIntermPlaneLoc) * (z > 0.) * (z <= h_port) * 1"
+Int_yOutputPlane.reduced_function(x,y,z) = " (x >= -w_port/2.) * (x <= w_port/2.) * (y >= YOutputPlaneLoc - dy/2.) * (y <= YOutputPlaneLoc) * (z > 0.) * (z <= h_port+th_gate) * 1"
+
+###############
```

Comment:
same as above. if this is the same as the Maurice case input with just the diagnostic added, could you edit the Maurice inputs with diagnostic inputs, instead of adding a new file.

Reply:
I can, but Andy and Jackie were not using the surface normal part, that's why I added it in a separate file. I will put this in the main input.

Reply:
Cool, adding this in the main input would be best! so we dont overload the examples directory :)

Reply:
Done